data-tx-hibernate6/build.gradle

```diff
@@ -0,0 +1,37 @@
+plugins {
+    id "io.micronaut.build.internal.data-module"
+}
+
+dependencies {
+    annotationProcessor mn.micronaut.inject.java
+    annotationProcessor mn.micronaut.graal
+
+    api projects.dataTx
+    api (libs.hibernate6.core) {
+        exclude group: 'org.jboss.spec.javax.transaction', module: 'jboss-transaction-api_1.2_spec'
+        exclude group: 'org.javassist', module: 'javassist'
+        exclude group: 'org.jboss:jandex', module: 'jar'
+    }
+
+    // TODO: Import hibernate6 when ready
+    api (mnSql.micronaut.hibernate.jpa) {
+        exclude module: 'data-tx-hibernate'
+    }
+
+    implementation mn.micronaut.inject
+
+    compileOnly mnSpring.spring.orm
+
+    testAnnotationProcessor mn.micronaut.inject.java
+    testAnnotationProcessor projects.dataProcessor
+
+    testImplementation projects.dataProcessor
+    // TODO: Separate tck for hibernate6?
+    //testImplementation projects.dataTck
+
+    testRuntimeOnly mnSql.h2
+    testRuntimeOnly (projects.dataHibernateJpa) {
```

Comment:
This will be projects.dataHibernate6Jpa in final step, if things go well till that step.